### PR TITLE
ForwardPayout should extend PayoutBase

### PR DESCRIPTION
**Pull request by hugohills-regnosys**

### CODE REVIEW

File: rosetta-source/src/main/rosetta/product-template-type.rosetta [hunk 1]
```diff
@@ -429,10 +429,14 @@ type ConstituentWeight: <"A class describing the weight of each of the underlier
...
 		then basketPercentage >= 0.0 and basketPercentage <= 1.0
 
-type ForwardPayout: <"Represents a forward settling payout. The 'Underlier' attribute captures the underlying payout, which is settled according to the 'SettlementTerms' attribute. Both FX Spot and FX Forward should use this component.">
+type ForwardPayout extends PayoutBase: <"Represents a forward settling payout. The underlier attribute captures the underlying payout, which is settled according to the settlementTerms attribute (which is part of PayoutBase). Both FX Spot and FX Forward should use this component.">
 	[metadata key]
+
 	underlier Product (1..1) <"Underlying product that the forward is written on, which can be of any type: FX, a contractual product, a security, etc.">
-	settlementTerms SettlementTerms (1..1) <"Settlement terms for the underlier that include the settlement date, settlement method etc.">
+
+	condition SettlementTerms: <"For foreign exchange contracts, the settlement the settlement terms must exist.">
```
Comment (lolabeis) [inline rosetta-source/src/main/rosetta/product-template-type.rosetta:437]: typo: "settlement" repeated in the description.

**Review by lolabeis — COMMENTED** [review]
What's the driver for the uptick in the validation failures?

**Comment by hugohills-regnosys** [comment]
> What's the driver for the uptick in the validation failures?

This is because `PayoutBase->payerReceiver` is mandatory, but not populated for the existing FpML FX samples.  The way existing FX samples are mapped would need to be reviewed in a future release.

File: rosetta-source/src/main/rosetta/product-template-type.rosetta (hunk 1 above)
Comment (hugohills-regnosys) [reply rosetta-source/src/main/rosetta/product-template-type.rosetta]: Fixed

File: RELEASE.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
 _What is being released?_
 
-This release updates the rosetta-dsl dependency:
+This release updates the `ForwardPayout` to extend `PayoutBase` to make it consistent with all other payouts.
 
-- Version upgrades included:
-  - 8.0.0: this version is a step towards the harmonisation of the expression syntax in rules and functions. No changes are required in the CDM. See release notes at https://github.com/REGnosys/rosetta-dsl/releases/tag/8.0.0
-  - 8.1.0: this version contains various patches to improve model validation while editing the model. No changes are required in the CDM. See release notes at https://github.com/REGnosys/rosetta-dsl/releases/tag/8.1.0
+The `settlementTerms` attribute has been removed from `ForwardPayout` as it an attribute of `PayoutBase`.
```
Comment (lolabeis) [inline RELEASE.md:7]: Typo: "is" missing

File: RELEASE.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
 _Review Directions_
 
-CDM Java implementors should update their maven pom.xml to the latest CDM maven artefact (groupId com.isda, artifactId cdm) and recompile.
+In the CDM Portal, select the Textual Browser and inspect each of the changes identified above.
+
+This release does not meaningfully change any existing mapping expectations:
```
Comment (lolabeis) [inline RELEASE.md:13]: rather than "meaningfully" (vague): "does not have any functional impact..."

File: RELEASE.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+
+- In the serialised JSON, the attribute ordering has changed due to the `settlementTerms` attribute change, however this has no meaningful impact on the representation of the model.
```
Comment (lolabeis) [inline RELEASE.md:15]: Also comment on the additional validation failures for FX samples due to their handling of payerReceiver

**Review by lolabeis — APPROVED** [review]

**Review by minesh-s-patel — APPROVED** [review]

**Pull request merged**